python/ray/autoscaler/autoscaler.py

```diff
@@ -256,6 +259,14 @@ def _update(self):
         for node_id in nodes:
             self.recover_if_needed(node_id, now)
 
+    def _node_resources(self, node_id):
+        instance_type = self.provider.node_tags(node_id).get(
+            TAG_RAY_INSTANCE_TYPE)
+        if instance_type:
+            return self.instance_types[instance_type].get("resources", None)
```

Comment:
```suggestion
            return self.instance_types[instance_type].get("resources", {})
```

Reply:
I think this works either way because updater.py will fill in an empty dict but sure